src/Hero6.UserInterface.SierraVga/Entities/Padding.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
```

Comment:
The header with licensing is missing, see how I structure all other code files for comparison